### PR TITLE
Update code owners

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Update Docs code owners

### CODE REVIEW

**Review by aevesdocker — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
 /docker-hub/ @jerae-duffin @craig-osterhout
 
-/engine/ @dockertopia @usha-mandya
+/engine/ @usha-mandya
 
-/registry/ @usha-mandya
+/subscription/ @jerae-duffin
 
-/single-sign-on/ @jerae-duffin 
+/docker-hub/publish/ @dvdklsn
```
Comment (aevesdocker) [inline .github/CODEOWNERS:18]:
```suggestion
/docker-hub/publish/ @dvdksn
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
 
-/single-sign-on/index.md @craig-osterhout
+/docker-hub/official_images/ @dvdklsn
```
Comment (aevesdocker) [inline .github/CODEOWNERS:20]:
```suggestion
/docker-hub/official_images/ @dvdksn
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -3,20 +3,20 @@
...
 
-/subscription/ @jerae-duffin
+/registry/ @dvdklsn
```
Comment (aevesdocker) [inline .github/CODEOWNERS:22]:
```suggestion
/registry/ @dvdksn
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (usha-mandya) [reply .github/CODEOWNERS]: Oops! Thanks @aevesdocker 😅

**Review by aevesdocker — APPROVED** [review]

**Pull request merged**